ui/packages/consul-peerings/app/components/consul/peer/form/chart.xstate.js

```diff
@@ -0,0 +1,26 @@
+export default {
+  id: 'consul-peer-form',
+  initial: 'generate',
+  on: {
+    INITIATE: [
+      {
+        target: 'initiate',
+      },
+    ],
+    GENERATE: [
+      {
+        target: 'generate',
+      },
+    ],
+    SUCCESS: [
```

Comment:
I don't see this state being used - is this a left over or do we plan to do something based on it?

Reply:
This is removed, probably a left over bit from a previous iteration 👍